Cart/common.css

```diff
@@ -4,20 +4,16 @@ body
     box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
     font-family: Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
 }
-a > .fa 
+.icon-ecomm a > .far , .fas
 {
```

Comment:
you can use beautify for proper indentation : )

Reply:
Thank You. I will make the changes